<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(150, 200, 250);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-1</y>
      <width>1021</width>
      <height>601</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>249</red>
          <green>244</green>
          <blue>206</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 244, 206);
border-radius:20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_ID">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiLight</family>
       <pointsize>9</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);
font: 75 9pt &quot;Cascadia Code SemiLight&quot;;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Login">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 221, 166);
border-radius:15px;</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Join">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>490</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕 semilight</family>
       <pointsize>10</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: 75 10pt &quot;맑은 고딕 semilight&quot;;
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code SemiLight</family>
       <pointsize>9</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom-color: rgb(0, 0, 0);
background-color: rgb(245, 245, 245);
font: 75 9pt &quot;Cascadia Code SemiLight&quot;;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>-50</y>
       <width>1021</width>
       <height>681</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 242, 186);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_MainLogo">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>210</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Code SemiBold</family>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25pt &quot;Cascadia Code SemiBold&quot;;
background: transparent;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>UPRIGHT</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_findID">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>490</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕 semilight</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
font: 75 10pt &quot;맑은 고딕 semilight&quot;;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>아이디 찾기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_findPW">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>490</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕 semilight</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
font: 75 10pt &quot;맑은 고딕 semilight&quot;;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>비밀번호 찾기</string>
      </property>
     </widget>
    </widget>
    <zorder>frame</zorder>
    <zorder>pushButton_Login</zorder>
    <zorder>lineEdit_Password</zorder>
    <zorder>lineEdit_ID</zorder>
    <zorder>pushButton_Join</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
